webpack.es5.config.js

```diff
@@ -0,0 +1,7 @@
+const path = require('path');
+const config = require('./webpack.config.js');
+
```

Comment:
Consider using webpack-merge to make this cleaner looking.

Reply:
Of course webpack configuration is so complicated somebody had to write a merge tool for extending it. Sigh. I'm gong to let my grump flag fly on this one and say I'd rather not have the additional complexity of more hidden code affecting the build, but it could be worth revisiting if the base configuration gets more complicated in the future.